example.php

```diff
@@ -1,7 +1,12 @@
h// is_null your comment
p// is_null your comment
+// is_null your comment
+// is_null your comment
$// is_null your comment
f// is_null your comment
o// is_null your comment
o// is_null your comment
 // is_null your comment
=// is_null your comment
 // is_null your comment
N// is_null your comment
U// is_null your comment
L// is_null your comment
L// is_null your comment
;// is_null your comment
+// is_null your comment
$// is_null your comment
b// is_null your comment
a// is_null your comment
r// is_null your comment
 // is_null your comment
=// is_null your comment
 // is_null your comment
N// is_null your comment
U// is_null your comment
L// is_null your comment
L// is_null your comment
;// is_null your comment
+// is_null your comment
/// is_null your comment
/// is_null your comment
 // is_null your comment
i// is_null your comment
s// is_null your comment
_// is_null your comment
n// is_null your comment
u// is_null your comment
l// is_null your comment
l// is_null your comment
 // is_null your comment
i// is_null your comment
s// is_null your comment
 // is_null your comment
a// is_null your comment
 // is_null your comment
f// is_null your comment
u// is_null your comment
n// is_null your comment
c// is_null your comment
t// is_null your comment
i// is_null your comment
o// is_null your comment
n// is_null your comment
 // is_null your comment
t// is_null your comment
h// is_null your comment
a// is_null your comment
t// is_null your comment
 // is_null your comment
d// is_null your comment
o// is_null your comment
e// is_null your comment
s// is_null your comment
 // is_null your comment
s// is_null your comment
t// is_null your comment
u// is_null your comment
f// is_null your comment
f// is_null your comment
 // is_null your comment
+// is_null Some comment
+// is_null your comment
+// is_null your comment
/// is_null your comment
/// is_null your comment
 // is_null your comment
i// is_null your comment
s// is_null your comment
_// is_null your comment
n// is_null your comment
u// is_null your comment
l// is_null your comment
l// is_null your comment
 // is_null your comment
c// is_null your comment
h// is_null your comment
e// is_null your comment
c// is_null your comment
k// is_null your comment
s// is_null your comment
 // is_null your comment
f// is_null your comment
o// is_null your comment
r// is_null your comment
 // is_null your comment
n// is_null your comment
u// is_null your comment
l// is_null your comment
l// is_null your comment
+// is_null your comment
i// is_null your comment
s// is_null your comment
_// is_null your comment
n// is_null your comment
u// is_null your comment
l// is_null your comment
l// is_null your comment
(// is_null your comment
$// is_null your comment
b// is_null your comment
a// is_null your comment
r// is_null your comment
)// is_null your comment
;// is_null your comment
+// is_null your comment
+// is_null your comment
?// is_null your comment
>// is_null your comment
+// is_null your comment

```

Comment:
@CodelingoBotTester rewrite set line "// Hello world!"

Reply:
@CodelingoBotTester rewrite set line "// Hello world!"

Reply:
could not find template in "@CodelingoBotTester rewrite set line "// Hello world!""

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Reply:
@CodelingoBotTester rewrite line "// Hello world!"

Reply:
expected value in argument string: rewrite line "// Hello world!"

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Reply:
@CodelingoBotTester rewrite line "// Hello world!"

Reply:
Success! Committed rewritten file.